spec/head_spec.js

```diff
@@ -1,10 +1,26 @@
-var head = require("../src/head.js")
+import { head } from '../src/head'
 
 describe('head', ()=> {
   it('returns the first element in an array', () => {
-    expect(head([1,2,3,4,5,])).toEqual(1)
-    expect(head([2,3,4,5,])).toEqual(2)
-    expect(head([3,4,5,])).toEqual(3)
+    const array = [1,2,3,4,5,]
+    expect(head(array)).toEqual(1)
+  })
+
+  it('returns the first element in an array', () => {
+    const array2 = [2,3,4,5,]
+    expect(head(array2)).toEqual(2)
+  })  
+
+  it('returns the first element in an array', () => {
+    const array3 = [3,4,5,]
+    expect(head(array3)).toEqual(3)
+  })
+
+  it('returns empty array when passed null', () => {
+    expect(head(null)).toEqual('this')
+  })
+
+  it('returns the first element in an array of objects', () => {
```

Comment:
I'm not sure I understand the purpose of this test? How does it differ from the first test?